app/controllers/to_do_lists_controller.rb

```diff
@@ -1,10 +1,11 @@
 class ToDoListsController < ApplicationController
 
   def index
-    lists = Finder.new(cookies[:filter]).perform
-    @lists_presentation = ListsPresentation.new lists
+    @lists_per_page = Finder.new(cookies[:filter]).perform
+    @lists_per_page = @lists_per_page.paginate(page: params[:page])
+    @lists_presentation = ListsPresentation.new @lists_per_page
 
-    @lists = lists.map { |list| SingleListPresenter.new list }
+    @lists = @lists_per_page.map { |list| SingleListPresenter.new list }
```

Comment:
@vlasiak, Can we expose `@` variable only for `@lists` and not for `@lists_per_page`?